examples/tutorials/api/estimators.py

```diff
@@ -0,0 +1,404 @@
+# `FluxPoints` are the represented by the "norm" scaling factor with
+# respect to the reference model:
+
+fp_result.norm.plot()
+plt.show()
+
+######################################################################
+# Dataset specific quantities ("counts like")
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+#
+# While the flux estimate and associated errors are common to all datasets,
+# the result also stores some dataset specific quantities, which can be useful
+# for debugging. The `~gammapy.maps.region.ndmap.RegionNDMap` allows for plotting of multidimensional data
+# as well, by specifying the primary `axis_name`:
+
+
+fp_result.counts.plot(axis_name="energy")
+plt.show()
+
+######################################################################
+#
+fp_result.npred.plot(axis_name="energy")
+plt.show()
+
+######################################################################
+#
+fp_result.npred_excess.plot(axis_name="energy")
+plt.show()
+
+######################################################################
+# Table conversion
+# ~~~~~~~~~~~~~~~~
+#
+# Flux points can be converted to tables:
+#
+
+table = fp_result.to_table(sed_type="flux", format="gadf-sed")
+display(table)
+
+######################################################################
+#
+table = fp_result.to_table(sed_type="likelihood", format="gadf-sed", formatted=True)
+display(table)
+
+######################################################################
+# A fully configured estimation
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+#
+# The following code shows fully configured flux points estimation:
+
+
+fit = Fit(
+    optimize_opts={"backend": "minuit"},
+    confidence_opts={"backend": "scipy"},
+)
+
+fp_estimator_config = FluxPointsEstimator(
+    source="crab",
+    energy_edges=energy_edges,
+    n_sigma=1,
+    n_sigma_ul=2,
+    selection_optional="all",
+    fit=fit,
+)
+
+print(fp_estimator_config)
+
+
+######################################################################
+#
+# %%time
+fp_result_config = fp_estimator_config.run(datasets=datasets)
+
+print(fp_result_config)
+
+######################################################################
+#
+fp_result_config.plot(sed_type="e2dnde", color="tab:orange")
+fp_result_config.plot_ts_profiles(sed_type="e2dnde")
+plt.show()
+
+
+######################################################################
+# Flux Map Estimation
+# -------------------
+
+dataset_cta = MapDataset.read(
+    "$GAMMAPY_DATA/cta-1dc-gc/cta-1dc-gc.fits.gz", name="cta_dataset"
+)
+
+plt.figure(figsize=(12, 6))
+counts_image = dataset_cta.counts.sum_over_axes()
+counts_image.smooth("0.05 deg").plot(stretch="linear", add_cbar=True)
+plt.show()
+
+######################################################################
+# Estimator configuration
+# ~~~~~~~~~~~~~~~~~~~~~~~
+#
+
+model = SkyModel(
+    spectral_model=PowerLawSpectralModel(), spatial_model=PointSpatialModel()
+)
+
+map_estimator = TSMapEstimator(
```

Comment:
The global section is about flux map estimation... Here, the TSMapEstimator is used, and later the FluxProfileEstimator. Should we present them quickly, summarized their features, before entering into the coding details?

Reply:
@AtreyeeS in my opinion if we make an API tutorial about estimators it is worth showing the `TSMapEstimator` again here.

Reply:
How about adding a link to the other notebook, where it is explained in more details?

Reply:
and here we could just read the results from the other notebooks